code/modules/clothing/spacesuits/void/excelsior.dm

```diff
@@ -50,3 +91,10 @@
 	)
 	helmet = /obj/item/clothing/head/space/void/excelsior
 	spawn_blacklisted = TRUE
+	auto_deploy_helmet = FALSE
+
+/obj/item/clothing/suit/space/void/excelsior/toggle_helmet()
+	if(is_excelsior(usr))
+		..()
+	else
+		to_chat(usr, SPAN_WARNING("The helmet fails to deploy, a built-in speaker says, \"Failed to locate implant, please contact your nearest Excelsior representative immediately for assistance\"."))
```

Comment:
Instead of blocking it here, you should rewrite code to don't show this HUD to people without implants. This change limits use if this suit as trophy, and other infiltration shenanigans, it should be done properly instead.

Reply:
Alright when I get off work I'll do the requested changes

Reply:
done